packages/components/src/components/combobox/combobox.styles.ts

```diff
@@ -0,0 +1,344 @@
+/**
+ * ---------------------------------------------------------------------
+ * 🔒 AUTOGENERATED BY VENDORISM
+ * Removing this comment will prevent it from being managed by it.
+ * ---------------------------------------------------------------------
+ */
+
+/* eslint-disable */
+import { css } from 'lit';
+
+export default css`
+	/* stylelint-disable */
+  :host {
+    display: block;
+  }
+
+  /** The popup */
+  .combobox {
+    flex: 1 1 auto;
+    display: inline-flex;
+    width: 100%;
+    position: relative;
+    vertical-align: middle;
+  }
+
+  .combobox::part(popup) {
+    z-index: var(--syn-z-index-dropdown);
+  }
+
+  .combobox[data-current-placement^='top']::part(popup) {
+    transform-origin: bottom;
+  }
+
+  .combobox[data-current-placement^='bottom']::part(popup) {
+    transform-origin: top;
+  }
+
+  /* Combobox */
+  .combobox__inputs {
+    flex: 1;
+    display: flex;
+    width: 100%;
+    min-width: 0;
+    position: relative;
+    align-items: center;
+    justify-content: start;
+    font-family: var(--syn-input-font-family);
+    font-weight: var(--syn-input-font-weight);
+    letter-spacing: var(--syn-input-letter-spacing);
+    vertical-align: middle;
+    overflow: hidden;
+    cursor: pointer;
```

Comment:
This should not be pointer when hovering over the text of the input field. When hovering on the input field, it should be `text` to indicate that we are using a regular input.

Reply:
the input field itself, now has cursor:text